src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,162 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_INITIAL_CAPACITY = 16;
+    private static final int RESIZE_FACTOR = 2;
+    private static final float LOAD_FACTOR = 0.75f;
+    private int size;
+    private int threshold;
+    private Node<K, V>[] table;
+
+    public MyHashMap() {
+        table = (Node<K, V>[]) new Node[DEFAULT_INITIAL_CAPACITY];
```

Comment:
Be attentive with the diamond operator in java. Use 'new Node<>()' instead of 'new Node<K,V>()' as the type is inferred.